src/app/pages/send/send-crypto-asset-form/form/stx/use-stx-send-form.tsx

```diff
@@ -37,9 +37,8 @@ export function useStxSendForm() {
   const { changeFeeByNonce } = useStacksValidateFeeByNonce(address);
 
   // get stx balance
-  const { filteredBalanceQuery } = useStxCryptoAssetBalance(address);
-  const { data: balance } = filteredBalanceQuery;
-  const availableBalance = balance?.availableUnlockedBalance ?? createMoney(0, 'STX');
+  const balance = useStxAddressBalance(address);
+  const availableBalance = balance.value?.stx.availableUnlockedBalance ?? createMoney(0, 'STX');
```

Comment:
I guess we do need to `createMoney(0` here to satisfy the form but maybe we can do better and not let them send if they have a real unknown balance.